hathor/manager.py

```diff
@@ -385,88 +384,91 @@ def _initialize_components(self) -> None:
             for tx in self.tx_storage.get_all_transactions():
                 tx.reset_metadata()
 
-        self.log.debug('load blocks and transactions')
```

Comment:
Where was this whole block moved to? If it's just removed, why don't we need it anymore?

Reply:
This block was mostly just simplified. But this diff is outdated.

Reply:
That's fine. I've already reviewed the new changes. Good job! Thanks!

Reply:
This was updated, the old method is now used fully unmodified when full-verification is enabled.